Library/Homebrew/cmd/install-vendor.sh

```diff
@@ -0,0 +1,108 @@
+source "$HOMEBREW_LIBRARY/Homebrew/utils/lock.sh"
+
+if [[ -n "$HOMEBREW_OSX" ]]
+then
+  Git_URL=""
+  Git_SHA=""
+  Ruby_URL=""
+  Ruby_SHA=""
+else
+  Git_URL=""
+  Git_SHA=""
+  Ruby_URL=""
+  Ruby_SHA=""
+fi
+
+cleanup-old() {
+  for version in "$HOMEBREW_LIBRARY/Vendor/$VENDOR_NAME/Versions"/*
```

Comment:
- `Library/Vendor` – Works for me (including the capitalized name for consistency with other directories in `Library`) and would help to keep the hierarchy flat.
- `Library/Homebrew/vendor` – Also works for me, but is a bit more buried and is somewhat less friendly to grepping all of `Library/Homebrew`.
- `opt/vendor` – Feels a bit out of place and I wouldn't want it to be confused with regular formulae that are symlinked there.